Update network-watcher-packet-capture-manage-cli.md

mrhoads opened the pull request:
I added a command for showing the Network Watcher extension for a Linux VM, as the difference in naming (e.g. AzureNetworkWatcherExtension for Linux) causes confusion when referencing the name of the extension on Windows.  Additionally, I may be mistaken, but I think the existing Step 1 under "Start a packet capture" is unnecessary.  "az network watcher show" is invalid.  Also, I changed "Start a packet capture" step 2 to retrieve the storage account with "az storage account list" instead of "azure storage account list".  Finally, I changed references to 'cmdlet' to 'command' to make it more consistent for the CLI portion of the documentation.

mrhoads committed:
Update network-watcher-packet-capture-manage-cli.md

I added a command for showing the Network Watcher extension for a Linux VM, as the difference in naming (e.g. AzureNetworkWatcherExtension for Linux) causes confusion when referencing the name of the extension on Windows.  Additionally, I may be mistaken, but I think the existing Step 1 under "Start a packet capture" is unnecessary.  "az network watcher show" is invalid.  Also, I changed "Start a packet capture" step 2 to retrieve the storage account with "az storage account list" instead of "azure storage account list".  Finally, I changed references to 'cmdlet' to 'command' to make it more consistent for the CLI portion of the documentation.

gurudennis commented on articles/network-watcher/network-watcher-packet-capture-manage-cli.md:
````diff
@@ -100,29 +106,21 @@ Once the preceding steps are complete, the packet capture agent is installed on
-```
-
-### Step 2
-
````
I believe this may actually be necessary because Step 3 of the current version of the document (where a packet capture is created) contains an inaccuracy. It most likely requires the --location parameter (which is currently missing) and thus finding the Network Watcher instance to use early on is in fact a good idea.

mrhoads replied:
Apologies that I'm not understanding this, but there is no `az network watcher show` command in the CLI.  For what's currently Step 1, you said that it's necessary, but by that, do you mean that a user should first run something like `az network watcher configure --resource-group NetworkWatcherRG --locations westcentralus --enabled` to create an instance before proceeding with identifying a storage account and then actually running `az network watcher packet-capture create`?

gurudennis replied:
I may be mistaken here (my experience with CLI is limited), but I noticed what seems to be an inconsistency. Specifically:

"az network watcher packet-capture show-status" is documented here as requiring the "--location" parameter to point to a specific Network Watcher resource.

"az network watcher packet-capture create" is documented here as not requiring such a parameter.

This inconsistency makes me believe that the latter is a bug, that is to say the parameter is actually required in both cases (this is certainly so in PowerShell, for example), but the documentation is flawed.

Hence, the proposed change to remove the prior retrieval of the Network Watcher instance may in fact be based on a faulty assumption. If that is the case, then the correct fix is to keep Step 1, and further add the "--location" parameter to the recommended "az network watcher packet-capture create" command line.

Does this make sense?

mrhoads replied:
I think this makes sense, though like you mentioned, `az network watcher packet-capture create` does not take --location.  Based on what you suggest for the correct fix, I re-added the original step 1 and added --location for `az network watcher packet-capture create` (assuming that adding --location to the CLI can and will be fixed separately)

gurudennis replied:
Oh, I think I understand now. You were right the first time around.

CLI intentionally deduces the location in the  packet-capture create call because it can (it has the VM resource ID). The query call however cannot deduce it because all it sees is the resource name.

Please proceed with your original change here, sorry about the confusion.

gurudennis commented on articles/network-watcher/network-watcher-packet-capture-manage-cli.md:
```diff
@@ -100,29 +106,21 @@ Once the preceding steps are complete, the packet capture agent is installed on
 
-### Step 3
+### Step 2
 
```
This doc could be written better. Let's take the opportunity to fix some low-hanging fruit.

When describing a formal step in a process, the first line of the description should usually be imperative, i.e. stating what to do next. Instead, this one opens up by talking about filters without establishing the context.

Let's add a sentence here to the effect of: "At this point you are ready to create a packet capture. First, let's examine the parameters that you may want to configure."

mrhoads committed:
establish context for adding filter

mrhoads committed:
re-added step 1

re-added Step 1 for az network watcher show and added --location to packet-capture create

mrhoads committed:
re-worked steps for starting a packet capture

this should match our exchange of comments; step 1 now goes back to the way it was originally in the PR and --location is removed from packet-capture create to match the CLI
